kyuubi-main/src/test/scala/org/apache/kyuubi/operation/datalake/IcebergOperationSuite.scala

```diff
@@ -19,8 +19,9 @@ package org.apache.kyuubi.operation.datalake
 
 import org.apache.kyuubi.config.KyuubiConf
 import org.apache.kyuubi.operation.{BasicIcebergJDBCTests, WithKyuubiServer}
-import org.apache.kyuubi.tags.DataLakeTest
+import org.apache.kyuubi.tags.{DataLakeTest, ExtendedSQLTest}
 
+@ExtendedSQLTest
```

Comment:
we can remove this tag to make the latter usage simple?

Reply:
ok